cmd/sfncli/sfncli.go

```diff
@@ -232,6 +232,9 @@ func tagsFromEnv() []*sfn.Tag {
 	if account := os.Getenv("_POD_ACCOUNT"); account != "" {
 		tags = append(tags, &sfn.Tag{Key: aws.String("pod-account"), Value: aws.String(account)})
 	}
+	if team := os.Getenv("_TEAM_OWNER"); account != "" {
+		tags = append(tags, &sfn.Tag{Key: aws.String("team"), Value: aws.String(account)})
```

Comment:
should this reference the `team` variable instead of `account`?

Reply:
I think this might fail to compile actually haha

Reply:
this is what i get for copy pasting